docs/csharp/language-reference/attributes/general.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "C# reserved attributes: Conditional, Obsolete, AttributeUsage"
-ms.date: 04/09/2020
+title: "C# reserved attributes: Miscellaneous"
+ms.date: 03/18/2021
 description: These attributes are interpreted by the compiler to affect the code generated by the compiler
```

Comment:
Since we've moved the attributes listed out of the title, should we add each of them to the description for SEO purposes?

Reply:
Good idea, done.